meson.build

```diff
@@ -982,6 +982,9 @@ if default_keymap == ''
 endif
 conf.set_quoted('SYSTEMD_DEFAULT_KEYMAP', default_keymap)
 
+default_timezone = get_option('default-timezone')
+conf.set_quoted('SYSTEMD_DEFAULT_TIMEZONE', default_timezone)
```

Comment:
so, there is quite an asymmetry here. SYSTEMD_DEFAULT_KEYMAP and SYSTEMD_DEFAULT_LOCALE both get applied in *absence* of any config. But this one here instead is about initializing the config with a default value, so it means in the end there *will* be a config.

That's quite a distinction, and I am not sure I like it.

I see no nice way to make SYSTEMD_DEFAULT_TIMEZONE work like the others admittedly though. After all, glibc never checks anything but /etc/localtime (well and $TZ, but that's nothing we can reasonably set, because it would disallow changingthe TZ later), and that's something we cannot really control ahead of time.

Hence maybe this is OK to add like this, but please rename this to something such as SYSTEMD_FIRSTBOOT_TIMEZONE or so, to indicate that this works differently from the others.

Reply:
I renamed it.

> But this one here instead is about initializing the config with a default value, so it means in the end there will be a config.

This is kind of done for all `sd-firstboot` stuff, this includes keymap and locale. If the default shouldn't be written there are 2 ways to prevent it:

- Don't `--prompt` the respective option
- Pass `-` to the respective prompt